### PR TITLE
Update the README file to include the information the sim_vim_sig.py …

### PR DESCRIPTION
**Pull request by Unique-Usman**
### Describe the changes you have made in this PR

Added the information about sim_vim_sig.py and how to run the script. 

_A clear and concise description of what you want to happen_

### Checklist

- [ ] Self-review of changed code
- [ ] Added automated tests where applicable
- [x] Update Docs & Guides

### CODE REVIEW

**Comment by Unique-Usman** [comment]
@etpeterson I added instruction on how to run sim_vim_sig.py for clearity.

File: phantoms/MR_XCAT_qMRI/README.md [hunk 1]
```diff
@@ -1 +1,61 @@
-# MR_XCAT_qMRI
\ No newline at end of file
+# 4D IVIM Phantom Generator
+
+A command-line tool for generating a 4D IVIM (Intravoxel Incoherent Motion) phantom as a NIfTI file.
+
+## Usage
+
+```sh
+python sim_vim_sig.py  [-h] [-b BVALUE [BVALUE ...] | -f FILE] [-n NOISE] [-m] [-i]
```
Comment (etpeterson) [inline phantoms/MR_XCAT_qMRI/README.md:8]: Could you provide an abbreviated version of this help in the scrip itself? Perhaps with a `-h` help argument or if the expected inputs aren't provided.

**Review by etpeterson — CHANGES_REQUESTED** [review]
Thanks for following up on this. Just the single comment.

File: phantoms/MR_XCAT_qMRI/README.md (hunk 1 above)
Comment (Unique-Usman) [reply phantoms/MR_XCAT_qMRI/README.md:8]: Do you meant this ? 

Screenshot from 2024-03-12 21-43-29


It is automatically, created.

Comment (etpeterson) [reply phantoms/MR_XCAT_qMRI/README.md:8]: Oh, excellent.

**Pull request merged**